File: moviejava/app/src/main/java/com/example/moviejava/extentions/ViewExtensions.java
package com.example.moviejava.extentions;

import android.widget.ImageView;
import androidx.databinding.BindingAdapter;
import com.bumptech.glide.Glide;
import com.example.moviejava.R;

import kotlin.jvm.JvmOverloads;


public class ViewExtensions {
    @JvmOverloads
    @BindingAdapter("url")
    public static void loadImage(ImageView img, String link) {
        Glide.with(img)
                .load("https://image.tmdb.org/t/p/w300"+link)
                .placeholder(R.drawable.is_loading)
                .into(img);
    }
    @JvmOverloads
    @BindingAdapter("url_mini")
    public static void loadImageMini(ImageView img, String link) {
        Glide.with(img)
                .load("https://image.tmdb.org/t/p/w200"+link)
                .placeholder(R.drawable.is_loading_mini)
                .into(img);
    }
}

File: movie/app/src/main/java/com/example/movie/extensions/ViewExtensions.kt
package com.example.movie.extensions

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.bumptech.glide.Glide
import com.example.movie.R

@BindingAdapter("url")
fun loadImage(img: ImageView, link:String?)
{
    Glide.with(img)
        .load("https://image.tmdb.org/t/p/w300${link}")
        .placeholder(R.drawable.is_loading)
        .into(img)
}
@BindingAdapter("url_mini")
fun loadImageMini(img: ImageView, link:String?)
{
    Glide.with(img)
        .load("https://image.tmdb.org/t/p/w200${link}")
        .placeholder(R.drawable.is_loading_mini)
        .into(img)
}